platform/platform.go

```diff
@@ -29,6 +26,8 @@ func Get() (*Platform, []string, error) {
 
 	p := &Platform{}
 	p.GoVersion = utils.GetString(platformInfo, "goV")
+	// Python version call was removed but we still want to report the field
+	// so as not to break code that depends on it.
```

Comment:
The line of code bellow is only used for the inventory project we can remove it and remove `PythonVersion` from the struct.
Once this is merged we can remove the corresponding code when we bump gohai in the agent code.